universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -93,16 +97,28 @@ public Content render(
 
   private Tag addFormEndpoints(Tag csrfTag, long programId, long blockId) {
     String blockCreateAction =
-        controllers.admin.routes.AdminProgramBlocksController.create(programId).url();
+        controllers.admin.routes.AdminProgramBlocksController.create(programId, Optional.empty())
+            .url();
     ContainerTag createBlockForm =
         form(csrfTag).withId(CREATE_BLOCK_FORM_ID).withMethod(POST).withAction(blockCreateAction);
 
+    String repeatedBlockCreateAction =
+        controllers.admin.routes.AdminProgramBlocksController.create(
+                programId, Optional.of(blockId))
```

Comment:
Suggestion (blocking): the repeater ID is an attribute of the resource being created rather than part of its resource handle. Please refactor this to use a hidden input for sending the repeater block's ID rather than making it part of the URL path.

Reply:
fixed!